__tests__/elements/Pagination/pagination.test.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { Pagination } from '../../../src'
+
+describe('<Pagination />', () => {
+  const itemsPerPage = 10
+  const totalItems = 124
+  const onClick = (page) => {}
+
+  it('renders Pagination component', () => {
+    const component = mount(<Pagination itemsPerPage={itemsPerPage}
+      totalItems={totalItems} onChange={onClick} />)
+
+    expect(component).toBeTruthy()
+    expect(component.find('.Pagination').length).toEqual(1)
+  })
+
+  it('renders Pagination component with specific link items', () => {
+    const maxPaginationSize = 9
+    const component = mount(<Pagination itemsPerPage={itemsPerPage}
+      totalItems={totalItems} maxPaginationSize={maxPaginationSize} onChange={onClick} />)
```

Comment:
in this case,
const newProps = Object.assign({}, props, { maxPaginationSize: 9})
const component = mount(<Pagination {...newProps} />)